tests/misc_env/install_prereq.py

```diff
@@ -158,21 +155,21 @@ def install_prereq(
             ceph.exec_command(cmd="sudo systemctl restart NetworkManager.service")
 
         if not skip_subscription:
-            if not setup_subscription_manager(ceph, "cdn"):
+            if not sm.setup_subscription_manager(ceph, "cdn"):
                 log.info("Trying to subscribe to stage server")
-                setup_subscription_manager(ceph, "stage")
+                sm.setup_subscription_manager(ceph, "stage")
 
-            status = subscription_manager_status(ceph)
+            status = sm.subscription_manager_status(ceph)
             if status == "Unknown" or skip_enabling_rhel_rpms:
                 log.info("Enabling local RHEL repositories")
                 if not setup_local_repos(ceph):
                     raise RepoConfigError("Failed to enable local RHEL repositories")
 
             elif enable_eus:
-                enable_rhel_eus_rpms(ceph, distro_ver)
+                SubscriptionManager.enable_rhel_eus_rpms(ceph, distro_ver)
 
             else:
-                enable_rhel_rpms(ceph, distro_ver)
+                SubscriptionManager.enable_rhel_rpms(ceph, distro_ver)
```

Comment:
incorrect call `SubscriptionManager`

Reply:
Thanks for pointing it out. Yes, since it's an instance method, I'll update it to use a SubscriptionManager object.